lib/GoogleSpreadsheetWorksheet.js

```
@@ -673,9 +673,35 @@ class GoogleSpreadsheetWorksheet {
     // https://developers.google.com/sheets/api/reference/rest/v4/spreadsheets/request#DeleteConditionalFormatRuleRequest
   }
 
-  async sortRange() {
-    // Request type = `sortRange`
-    // https://developers.google.com/sheets/api/reference/rest/v4/spreadsheets/request#SortRangeRequest
+  // sortSpec is of the format
+  // [[rowIndex, order], [rowIndex, order]...] where order is 'desc' or 'asc'
+  // https://developers.google.com/sheets/api/reference/rest/v4/spreadsheets/request#SortRangeRequest
+  async sortRange(sortSpecs, range) {
+    if (!sortSpecs.length) return null;
```

maybe better to throw an error?